### PR TITLE
Creation of Polish translation

### PR DESCRIPTION
**Pull request by Greg-21**
As written in the title, in this pull request I created Polish translation.

### CODE REVIEW

File: src/main/resources/assets/bingolobby/lang/pl_pl.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "bingolobby.countdown.minutes": "Gra rozpocznie się za %s min.",
+  "bingolobby.countdown.set": "Rozpoczęto odliczanie %s min i %s sek.",
+  "bingolobby.scoreboard.online": "Graczy online: %s",
+  "bingolobby.scoreboard.no_joining": "Drużyny będą wkrótce otwarte.",
+  "bingolobby.scoreboard.players_per_team": "Graczy na drużynę: %s",
+  "bingolobby.scoreboard.status.info": "Status: %s",
+  "bingolobby.scoreboard.status.player": "Gracz",
+  "bingolobby.scoreboard.status.vip": "VIP",
+  "bingolobby.scoreboard.noteam": "Brak drużyny",
+  "bingolobby.scoreboard.nogame": "Brak utworzonej gry",
```
Comment (Greg-21) [inline src/main/resources/assets/bingolobby/lang/pl_pl.json:22]: I think that this should be changed to "No created game" in the U.S. English translation file as the word "active" is not used in any Bongo command and may be confusing.

**Review by Greg-21 — COMMENTED** [review]
>

File: src/main/resources/assets/bingolobby/lang/pl_pl.json (hunk 1 above)
Comment (Greg-21) [inline src/main/resources/assets/bingolobby/lang/pl_pl.json:22]: I think that this should be changed to "No created game" in the U.S. English translation file as the word "active" is not used in any Bongo command and may be confusing.

**Review by Greg-21 — COMMENTED** [review]
I would like to discuss one thing.

File: src/main/resources/assets/bingolobby/lang/pl_pl.json (hunk 1 above)
Comment (noeppi-noeppi) [reply src/main/resources/assets/bingolobby/lang/pl_pl.json:22]: Yeah. It's called active in the code but when you reworked the translations it got changed I think. You can just change it here and I'll merge

Comment (Greg-21) [reply src/main/resources/assets/bingolobby/lang/pl_pl.json:22]: Oh yeah, I did so for consistency with the `/bingo create` command.

**Pull request merged**